package.json

```diff
@@ -32,5 +32,15 @@
     "sinon": "^1.10.3",
     "sinon-as-promised": "^1.2.0",
     "time-grunt": "^1.0.0"
+  },
+  "homepage": "https://github.com/ReadyTalk/cukefarm",
+  "keywords": [
+    "cucumber",
+    "cucumberjs",
+    "protractor",
+    "selenium"
+  ],
+  "scripts": {
+    "test": "./node_modules/grunt-cli/bin/grunt"
```

Comment:
nit: Any executables defined by dependency modules are automatically added to the PATH used for executing scripts in the package.json file. So specifying just `"test": "grunt"` would also work.